deployment/.env.example

```diff
@@ -3,7 +3,7 @@
 PROJ_PREFIX=myproj
 COMPOSE_PROJECT_NAME=${PROJ_PREFIX}
 
-#MYW_DB_NAME=iqgeo
+#MYW_DB_NAME=myproj
 
```

Comment:
```suggestion
```

Because this value is defined in the [docker-compose.yml](/deployment/docker-compose.yml) it isn't necessary to define/override in typical use, and a typical deployment (with the exception of on-premise/local deployments) wouldn't ever use the values from this file, I think it can just be removed entirely to avoid any possible confusion/mistake.